composer.json

```diff
@@ -41,7 +118,49 @@
 		"sirbrillig/phpcs-changed": "2.10.0",
 		"spatie/phpunit-watcher": "^1.23",
 		"yoast/phpunit-polyfills": "^1.0",
-		"composer/installers": "^2.2"
+		"composer/installers": "^2.2",
+		"wpackagist-plugin/akismet": "*",
+		"wordpress-plugin/bbpress": "2.6.*",
+		"wpackagist-plugin/campt-indian-payment-gateway": "*",
+		"wordpress-plugin/camptix-bd-payments": "1.2",
+		"wordpress-plugin/camptix-mercadopago": "1.0.6",
+		"wpackagist-plugin/camptix-pagseguro": "*",
+		"wpackagist-plugin/camptix-payfast-gateway": "*",
+		"wpackagist-plugin/camptix-trustcard": "*",
+		"wpackagist-plugin/camptix-trustpay": "*",
+		"wordpress-plugin/camptix-paystack": "1.0.0",
+		"wordpress-plugin/camptix-paynow": "1.0.4",
+		"wpackagist-plugin/classic-editor": "dev-trunk",
+		"wpackagist-plugin/custom-content-width": "*",
+		"wpackagist-plugin/edit-flow": "*",
+		"wpackagist-plugin/email-post-changes": "dev-trunk",
+		"wpackagist-plugin/gutenberg": "*",
+		"wpackagist-plugin/hyperdb": "dev-trunk",
+		"wpackagist-plugin/jetpack": "*",
+		"wpackagist-plugin/liveblog": "*",
+		"wpackagist-plugin/public-post-preview": "*",
+		"wpackagist-plugin/pwa": "*",
+		"wpackagist-plugin/supportflow": "dev-trunk",
+		"wpackagist-plugin/tagregator": "dev-trunk",
+		"wpackagist-plugin/wordpress-importer": "*",
+		"wpackagist-plugin/wp-cldr": "*",
+		"wpackagist-plugin/wp-super-cache": "*",
+		"wordpress-meta/wporg-profiles-wp-activity-notifier": "1.1",
+		"wporg/wporg-mu-plugins": "dev-build",
+		"wpackagist-theme/p2": "*",
+		"wpackagist-theme/twentyten": "*",
```

Comment:
I wonder if all of these are necessary anymore.

Reply:
I think they are because this is how https://wpackagist.org/ works (see examples)
Or you're suggesting that these themes with outdated versions, like 2010 here, aren't really needed?

Regarding the packages above and the versions listed here, I think one of them could be removed though, since the versions are already detailed in the packages section.

Reply:
> Or you're suggesting that these themes with outdated versions, like 2010 here, aren't really needed?
Yes.

It's fine either way but may save on bandwidth and requests if we can eliminate the old ones.

Reply:
Are we not talking about the development environment specifically? If someone wanted to work on an old theme, could they not just install it manually?

Reply:
It looks like we do still provide Twenty Ten+ for new sites, so if we aim to match production, we should install it. However, as long as none of the test content uses the old themes, I'd be okay with omitting them. If an issue pops up on an older theme, we can install it to debug/fix.

> Do we have a policy in place specifying which version we support?

I don't think so, officially? But I'm not sure. Ideally we're not making anything unusable on the older themes, but in my past work I wasn't going through each Twenty** and testing it— usually the last 2 or 3 themes.

Reply:
Alright, thanks. I'll omit until only the three latest remain. There's little need to deal with old themes in general, and for who need them will know how to install them.